app/helpers/application_helper.rb

```diff
@@ -11,7 +11,7 @@ def title(page_title)
   def application_meta_tags
     [
       tag('meta', :name => 'app-environment', :content => Rails.env),
-      tag('meta', :name => 'app-config-mapbox-id', :content => ENV['MAPBOX_ID'])  
+      tag('meta', :name => 'app-config-mapbox-id', :content => ENV['MAPBOX_ID'])
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.